extensions/amp-image-lightbox/0.1/test/test-amp-image-lightbox.js

```diff
@@ -112,6 +106,52 @@ describes.realWin(
       );
     });
 
+    it('should render correctly with an img element', async () => {
+      const lightbox = await getImageLightbox();
+      const impl = await lightbox.getImpl(false);
+
+      const noop = () => {};
+      impl.getViewport = () => ({
+        onChanged: noop,
+        enterLightboxMode: noop,
+      });
+      impl.getHistory_ = () => ({
+        push: () => Promise.resolve(),
+      });
+      impl.enter_ = noop;
+
+      const img = doc.createElement('img');
+      img.setAttribute('src', 'data:');
+      impl.open_({caller: img});
+
+      const container = lightbox.querySelector(
+        '.i-amphtml-image-lightbox-container'
+      );
+      expect(container).to.not.equal(null);
+
+      const caption = container.querySelector(
+        '.i-amphtml-image-lightbox-caption'
+      );
+      expect(caption).to.not.equal(null);
+      expect(caption).to.have.class('amp-image-lightbox-caption');
+
+      const viewer = container.querySelector(
+        '.i-amphtml-image-lightbox-viewer'
+      );
+      expect(viewer).to.not.equal(null);
+
+      const image = viewer.querySelector(
+        '.i-amphtml-image-lightbox-viewer-image'
+      );
+      expect(image).to.not.equal(null);
```

Comment:
nit for meaningful syntax
```suggestion
      expect(container).to.not.be.null;

      const caption = container.querySelector(
        '.i-amphtml-image-lightbox-caption'
      );
      expect(caption).to.not.be.null;
      expect(caption).to.have.class('amp-image-lightbox-caption');

      const viewer = container.querySelector(
        '.i-amphtml-image-lightbox-viewer'
      );
      expect(viewer).to.not.be.null;

      const image = viewer.querySelector(
        '.i-amphtml-image-lightbox-viewer-image'
      );
      expect(image).to.not.be.null;
```